src/elements/content-uploader/ItemAction.js

```diff
@@ -66,6 +85,18 @@ const ItemAction = ({ status, onClick, intl, isResumableUploadsEnabled, isFolder
             break;
     }
 
+    if (STATUS_ERROR && code === ERROR_CODE_UPLOAD_FILE_SIZE_LIMIT_EXCEEDED && !!onFreemiumUpsellCTAClick) {
+        return (
```

Comment:
We generally try to avoid multiple returns, as it can lead to confusing behavior. Can we tie into the status switch above and the existing `onClick` handler, instead?

Reply:
The PrimaryButton design (a new resin tag, no tooltip) is quite different from the current Plain Button icon with tooltip. Classname `bcu-item-action` also limits the width and height of this component which would cause the PrimaryButton to be cut off or misaligned. In addition, code clean up may be easier in the future.